yellow-paper/docs/decentralisation/governance.md

```diff
@@ -0,0 +1,237 @@
+
+```mermaid
+sequenceDiagram
+
+participant Current Canonical Rollup as Current Rollup
+participant Sequencers
+participant Version Registry as Governance
+participant Next Rollup
+participant Anyone
+
+Current Canonical Rollup ->> Version Registry: proposeCanonicalRollup(nextAddress)
+Note right of Version Registry: Vote starts in N days, eg 7
+Anyone ->> Version Registry: delegateTo(otherAddress)
+Anyone ->> Current Canonical Rollup: delegateTo()
+Note right of Version Registry: Must be delegated before vote starts 
+loop Voting
+    loop Canonical Rollup Voting
+        Sequencers ->> Current Canonical Rollup: canonicalVote(nextAddress, yes | no, amount)
+        Current Canonical Rollup --> Current Canonical Rollup: Count votes
+    end
+    Current Canonical Rollup ->> Version Registry: publishVoteResult(yes | no | abstain)
+    Anyone ->> Version Registry: addVote(yes | no | abstain)
+    Version Registry --> Version Registry: Count votes
+end
+Note right of Version Registry: Vote passed!
+Version Registry ->> Version Registry: markPendingCanonical(nextAddress)
+Note right of Version Registry: Wait at least 30 days!
+Next Rollup ->> Version Registry: markCanonical(nextAddress)
+Sequencers ->> Next Rollup: Proposing new blocks here!
+```
+
+### Emergency mode
+Emergency mode is proposed to be introduced to the initial instance "v0" or "v1" of Aztec, whatever the first instance or deployment is. Emergency mode **will not be included as part of the canonical governance contracts or registry**. If future deployments wish to have a similar security council, they can choose to do so. In this design, the current rollup can determine the timelock period as articulated above, within some predefined constraints, eg 3-30 days. Explicitly, the current rollup can give a security council the ability to define what this timelock period may be, and in the case of a potential vulnerability or otherwise, may be well within it's rights to choose the smallest value defined by the immutable governance conract to ensure that the network is able to recover and come back online as quickly as possible. 
+
+![Emergency Mode Image](../decentralisation/images/Aztec-Governance-Summary-4.png)
+
+#### Unpausing by default
+In the first instance, it's expected that this security council can _only_ pause the rollup instance, not make any other changes to the instance's functionaiity. It is important that after N days (eg 180), or after another rollup has been marked canonical and Y days (eg 60), this rollup _must_ become unpaused eventually - otherwise it's pratcially bricked from the perspective of those users choosing immutable portals, and could leave funds or other things beloning to users (eg identity credentials or something wacky) permanently inside of it. The same is true for all future instances that have pausable functionalities.
```

Comment:
```suggestion
In the first instance, it's expected that this security council can _only_ pause the rollup instance, not make any other changes to the instance's functionality. It is important that after N days (eg 180), or after another rollup has been marked canonical and Y days (eg 60), this rollup _must_ become unpaused eventually - otherwise it's practically bricked from the perspective of those users choosing immutable portals, and could leave funds or other things belonging to users (e.g., identity credentials or something wacky) permanently inside of it. The same is true for all future instances that have pause functionalities.
```